src/rules/dollar-variable-default/__tests__/index.js

```diff
@@ -0,0 +1,58 @@
+import rule, { ruleName, messages } from "..";
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  syntax: "scss",
+
+  accept: [
+    {
+      code: "$var: 10px !default",
+      description: "Global variable with !default"
+    },
+    {
+      code: "a { $var: 10px !default }",
+      description: "Local variable with !default"
+    }
+  ],
+
+  reject: [
+    {
+      code: "$var: 10px",
+      message: messages.expected,
+      description: "Global variable without !default"
+    },
+    {
+      code: "a { $var: 10px }",
```

Comment:
oh yeah, could you add `line` and `column` numbers to the rejected messages just to make sure that the error is shown at the correct place (`$`-sign)?

Like here:
https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/double-slash-comment-inline/__tests__/index.js#L43-L52